components/prism-cmake.js

```diff
@@ -0,0 +1,8 @@
+Prism.languages.cmake = Prism.languages.extend("python", {
```

Comment:
Why did you choose to extend Python?

I'm not very familiar with the CMake syntax but they don't seem to be that much alike.

(Also, single quotes please.)

Reply:
Two reasons:
1. # comments
2. parenthesis
I haven't found any generic script-like syntax so I chose the simplest possible syntax :)

Reply:
So, the only things we need from Python are comments, strings, and punctuation. Is that right?

In that case we could just copy them over from Python.

Reply:
Ok

Reply:
Could you give me a hand with punctation?
We need ( ) $< >. $< is a generator expression, so we have to match those two characters, not only <

Reply:
You probably want `${...}` as well, right?
If so: `/[()>}]|\$[<{]/`

Reply:
Yes, that's right. Thank you!